cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -186,10 +186,62 @@ The following is the generic procedure for Major/Minor version upgrades:
 
 .. _cloudera-release-specific-upgrade-notes:
 
+.. rubric:: Upgrading CDAP 3.0 to 3.1 and Upgrading CDH 5.3 to 5.4
+
+**Background:** CDH 5.3 ships with Hbase 0.98 while CDH 5.4 ships with HBase 1.0. We support
+CDH 5.4 as of CDAP 3.1.0. Upgrading from CDH 5.3 to CDH 5.4 entails an HBase upgrade in
```

Comment:
entails an Hbase upgrade sounds incorrect, it includes an HBase upgrade by default , while CDAP upgrade is done manually by the user.. meaning they are two different steps. the sentence is a little confusing.